### PR TITLE
ci(docker): get rid of trezor-suite gitlab image

### PR DESCRIPTION
**Pull request by karliatto**
## Description
When running CI in GitLab we were using docker image "registry.gitlab.com/satoshilabs/trezor/trezor-suite/base:latest" but now it is not needed anymore. 
So with this PR I am getting rid of that image and simplifying a bit the process.

### CODE REVIEW

**Review by vdovhanych — COMMENTED** [review]
Please change this to all of the testing flows not only connect. Its being used for everything and i would like to change it in one go.

File: docker/docker-connect-test.sh [hunk 1]
```diff
@@ -47,9 +44,6 @@ FIRMWARE=""
 FIRMWARE_URL=""
 INCLUDED_METHODS=""
 EXCLUDED_METHODS=""
-DOCKER=true
-# TODO: DOCKER_PATH appears unused. Remove or export?
-DOCKER_PATH="docker"
```
Comment (mroz22) [inline docker/docker-connect-test.sh]: these are used, you can run tests from command line in one script. and it either starts trezor-user-env or not.

**Comment by karliatto** [comment]
> Please change this to all of the testing flows not only connect. Its being used for everything and i would like to change it in one go.

@vdovhanych I cannot find the image "registry.gitlab.com/satoshilabs/trezor/trezor-suite/base:latest" anywhere else in the repo but maybe I am missing somethig.

File: docker/docker-connect-test.sh (hunk 1 above)
Comment (karliatto) [reply docker/docker-connect-test.sh]: Ok, I did not think about use case where you want to run this script locally with trezor-user-env locally. I will keep it.

**Comment by vdovhanych** [comment]
> > Please change this to all of the testing flows not only connect. Its being used for everything and i would like to change it in one go.
> 
> @vdovhanych I cannot find the image "registry.gitlab.com/satoshilabs/trezor/trezor-suite/base:latest" anywhere else in the repo but maybe I am missing somethig.

Hmm, I thought it's used for e2e testing also, but it's actually using a cypress image, right? Well, if it's not anywhere else, then it's good, I guess :).

File: docker/docker-connect-test.sh (hunk 1 above)
Comment (karliatto) [reply docker/docker-connect-test.sh]: Done, by those fixups 
* https://github.com/trezor/trezor-suite/pull/14178/commits/90aef7972ae1681c8caa66420dcbcce749d357ab
* https://github.com/trezor/trezor-suite/pull/14178/commits/69ec61cd7459065dc65b84e3a7562c2e1d30361d

Not that functionality of the script is back.

**Comment by karliatto** [comment]
It looks like all tests are now passing here.

**Review by vdovhanych — APPROVED** [review]

**Pull request merged**